.github/workflows/evals-model.yml

```
@@ -1,32 +1,78 @@
+      FOREST_EVAL_ROLE: ${{ inputs.affected_role }}
+      FOREST_EVAL_DISABLE_AGENTIC_PLANNER: ${{ inputs.disable_agentic_planner && '1' || '0' }}
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: select pull-request eval surface
+        if: github.event_name == 'pull_request'
+        env:
+          BASE_SHA: ${{ github.event.pull_request.base.sha }}
+          HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+        run: echo "FOREST_EVAL_ROLE=$(python3 evals/scripts/affected_role.py --base \"$BASE_SHA\" --head \"$HEAD_SHA\")" >> "$GITHUB_ENV"
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:74: Double quote to prevent globbing and word splitting

(shellcheck)